docs/widgets.md

```
@@ -374,8 +384,9 @@ ArtisanalApp(
 |-----------|---------|-------------|
 | `enableRenderMetrics` | `true` | Subscribe to `RenderMetricsMsg` from TUI runtime |
 | `enableRenderMetricsInjection` | `true` | Listen to `RenderMetricsInjector` stream |
-| `debugOverlay` | `false` | Show runtime debug overlay on startup |
-| `debugOverlayPosition` | `topRight` | Where the overlay appears |
+The former `WidgetApp.debugOverlay` and `debugOverlayPosition` options were
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)